### ISSUE TITLE
Certs and stuff

### ISSUE THREAD

**Issue by gonzalo-** [conformal/xombrero#128]
Hi guys,

You see this https://rya.nc/https-script.html ?

Thanks

**Comment by marcopeereboom** [comment]
Yep i saw but it hasn’t been picked up by webkit1 it seems.  This is a dep for us so we don’t control that.
